src/client/client4.js

```diff
@@ -2269,6 +2269,13 @@ export default class Client4 {
         );
     };
 
+    testSiteURL = async (siteURL) => {
+        return this.doFetch(
+            `${this.getBaseRoute()}/site_url/test`,
+            {method: 'post', body: siteURL}
```

Comment:
Will probably change once the server is updated per comment - https://github.com/mattermost/mattermost-server/pull/11944/files#r318155227